integrations/sui/sources/dwallet_cap.move

```diff
@@ -0,0 +1,47 @@
+module dwallet_network::dwallet_cap {
+    use std::vector;
+    use sui::object::{Self, ID, UID};
+    use sui::event;
+    use sui::tx_context::TxContext;
+
+
+    struct DWalletCap has key, store {
+        id: UID,
+        dwallet_network_cap_id: ID,
+    }
+
+
+    struct DWalletNetworkInitCapRequest has copy, drop {
+        cap_id: ID,
+        dwallet_network_cap_id: ID,
```

Comment:
Note that here we're using the same type for both IDs, wheras one of them comes from a different network. This could cause confusion and problems in the future. Perhaps a better idea would be to import the dwallet Network crate and use its ID. For now we can leave it as-is, but at least document this so the assumptions and coupling will be explicitly mentioned